src/CLI/Program.cs

```diff
@@ -19,7 +20,7 @@ class Program
         private readonly IBrowserAbstraction _browserAbstraction;
         private readonly IScanDelay _scanDelay;
 
-        private ScanResults _scanResults;
+        private IReadOnlyCollection<ScanResults> _scanResults;
```

Comment:
For consistent naming, perhaps `_scanResultsCollection` ? This could be a separate PR if that makes sense